local-volume/helm/provisioner/templates/provisioner-cluster-role-binding.yaml

```diff
@@ -14,6 +14,16 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
 ---
 apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: local-storage-provisioner-node-binding-clusterrole
+  namespace: {{ .Values.common.namespace }}
+rules:
+- apiGroups: [""]
```

Comment:
Just curious, is setting apiGroups to "" same as "*"? Or does it restrict things further?

Reply:
```
apiGroups: [""] # "" indicates the core API group
```
From: https://kubernetes.io/docs/admin/authorization/rbac/